boards/arm/nucleo_f446re/nucleo_f446re.yaml

```diff
@@ -14,5 +14,5 @@ supported:
   - gpio
   - spi
   - i2c
-ram: 96
+ram: 128
```

Comment:
In that case, SoC sram is split in 2 areas (SRAM1 112K and SRAM2 16K), so this is correct.

Reply:
But in [stm32f446Xe.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/st/f4/stm32f446Xe.dtsi#L12) it is defined as 128k...